backend/siarnaq/api/episodes/models.py

```diff
@@ -298,7 +298,7 @@ def initialize(self):
         self.external_id_private = f"private_{self.external_id_public}_{key}"
 
         teams = self.get_eligible_teams()
-        for is_private in [False, True]:
+        for is_private in [True, False]:
```

Comment:
Just wondering, was there a reason for swapping them around?

Reply:
the overall tour workflow is basically "do private first then public!". and same for code.
When I was testing, i was looking at IDs, seeing private IDs be _higher_ than public ones. This goes against "private first then public" and so it gave me a headache and i didn't like it